udata/core/post/models.py

```diff
@@ -38,6 +45,9 @@ class Post(db.Datetimed, db.Document):
     owner = db.ReferenceField('User')
     published = db.DateTimeField()
 
+    body_type = db.StringField(
+        choices=list(BODY_TYPES), default='markdown', required=False)  # should we?
```

Comment:
Should it be required?
```suggestion
        choices=list(BODY_TYPES), default='markdown', required=False)
```